README.md

```diff
@@ -4,33 +4,21 @@
 
 ---
 
-A collection of templates:
-
-* **[CoroutineTemplate](https://github.com/nimblehq/android-templates/tree/kotlin/CoroutineTemplate)**
-* **[RxJavaTemplate[DEPRECATED]](https://github.com/nimblehq/android-templates/tree/kotlin/RxJavaTemplate)**
+Our Android template: **[CoroutineTemplate](https://github.com/nimblehq/android-templates/tree/kotlin/CoroutineTemplate)**
 
 ## Setup
 
 1. Clone or download this repository to your local machine, then extract and open the folder
-2. Run `newproject.sh` script to create a new project with the following inputs:
-
+2. Install [Kscript](https://github.com/holgerbrandl/kscript#installation)
+3. Run `kscript new_project.kts` to create a new project with the following arguments:
+```   
+  package-name=                      New package name (i.e. com.example.package)
+  app-name=                          New app name (i.e. MyApp, "My App")
 ```
```

Comment:
please correct indentation

```suggestion
    ```   
      package-name=                      New package name (i.e. com.example.package)
      app-name=                          New app name (i.e. MyApp, "My App")
    ```
```

Reply:
Hmmm, really? We're using the same indentation as before 🙏

Reply:
@Tuubz we should align this code block to match to the `3` section in the list only, right? https://nimblehq.co/compass/intro/contribute/#markdown

Reply:
Aha, I see! Thanks for the clarification 🥇